e2e-tests/tests/init-deploy/00-deploy-operator.yaml

```diff
@@ -1,3 +1,14 @@
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: init-deploy-resource-quota
+spec:
+  hard:
+    requests.cpu: "2"
+    requests.memory: "4Gi"
+    limits.cpu: "4"
+    limits.memory: "6Gi"
```

Comment:
Increased memory in order to address some errors as reported here: https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-811/21e66a83/init-deploy.log

e.g. 

```
    logger.go:42: 18:50:03 | init-deploy | 2025-01-24 18:43:39 +0000 UTC	Warning	StatefulSet.apps init-deploy-orc		FailedCreate	create Pod init-deploy-orc-1 in StatefulSet init-deploy-orc failed error: pods "init-deploy-orc-1" is forbidden: exceeded quota: init-deploy-resource-quota, requested: limits.memory=1Gi, used: limits.memory=2560Mi, limited: limits.memory=3Gi	statefulset-controller	

```

Reply:
commit: https://github.com/percona/percona-server-mysql-operator/pull/811/commits/7a0e69755f826a0bdf105b7641001dfb2c6dbe2f